src/components/shared/StackSettings.css

```diff
@@ -6,8 +6,8 @@
   display: flex;
   height: 25px;
   flex-flow: row nowrap;
+  align-items: stretch;
   padding: 0;
-  line-height: 25px;
```

Comment:
The line-height was being used to center text and inline elements vertically within the settings bar, but `align-items: center` (which is already set on `.panelSettingsListItem`) is a better way to do that.

Reply:
sounds good to me!